tensorflow_serving/example/resnet_warmup.py

```diff
@@ -12,51 +12,73 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""Creates the tf_serving_warmup_requests file to warm up a server.
+"""Creates the tf_serving_warmup_requests file to warm up a ResNet SavedModel.
    
-   1. Invoke this script to generate the tf_serving_warmup_requests file.
-   2. Create a directory 'assets.extra' in the folder of the saved model.
-   3. copy the file tf_serving_warmup_requests into that folder.
-   4. Restart tensorflow_model_server.
+   1. Invoke this script passing in the model directory (including version
+        folder) as an argument.
+   2. Restart tensorflow_model_server.
 
-   If unsure where the file goes, look for the output:
+   If unsure of the model directory, look for the output:
    'No warmup data file found at' in the tensorflow_model_server
    startup log
 
-   When copied to the proper location look for the output:
+   After the script is run, and tensorflow_model_server is restarted, to verify
+   it is working look for the output:
    'Starting to read warmup data for model at' in the tensorflow_model_server
    startup log
 
    Usage example:
-     python resnet_warmup.py
+     python resnet_warmup.py model_dir
```

Comment:
Probably worth being more specific with saved_model_dir. Also, worth commenting saved_model_dir should be the one containing the saved_model.pb file.